frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1026,7 +1026,7 @@
                             vm.message = details;
                             vm.showClock = true;
                             vm.countDownTimer = function() {
-                                vm.remainingTime = vm.message.remaining_time;
+                                vm.remainingTime = Math.floor(vm.message.remaining_time);
```

Comment:
Just curious: When the string is not a number then `Math.floor()` will return 0 or something else?

Reply:
My bad, it will return NaN only.